src/estimators/vector_of_counts_sketch_operator.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sketch operator for Vector-of-Counts."""
+import copy
+
+
+class SketchOperator:
+  """Sketch operations for supporting frequency dedupe.
+
+  The operator defined in this class is NOT intended to be used as a generic
```

Comment:
grammar: The operators defined in this class are NOT intended to be used ....

Reply:
Done.